src/client/datascience/common.ts

```diff
@@ -110,17 +110,19 @@ export function traceCellResults(prefix: string, results: ICell[]) {
     }
 }
 
+const jupyterLanguageToMonacoLanguageMapping = new Map([
+    ['c#', 'csharp'],
+    ['f#', 'fsharp'],
+    ['q#', 'qsharp'],
+    ['c++11', 'c++'],
+    ['c++12', 'c++'],
+    ['c++14', 'c++']
```

Comment:
```suggestion
    ['c++14', 'c++'],
    ['R', 'r'],
    ['Rust', 'rust'],
    ['julia', 'julia'],
    ['powershell', 'powershell'],
    ['python', 'python']
```

I was already doing this in a namespace. But now that you have it here, please add these languages so we can always call this.
Also, I'm not sure 'julia' is correct for Monaco, can you check please?

Reply:
Not sure we need `, rust, julia, powershell & python`. They are identitcal.
If you look at the code, we just lowercase them. Hence not necessary to add to the dict, as we'll get the same behavior.

Reply:
We didn't need them for the code to work, but its good to explicitly mention the languages we support in code.
A case might happen where the monaco name is capitalized and the kernel name is not, we'd need to add that case, in contrast to cases like r, or rust. I'd rather have them all listed.